docs/api/transaction/get-raw-transaction.schema.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "GET raw transaction",
+  "title": "GetRawTransactionResult",
+  "type": "object",
+  "required": ["raw_tx"],
+  "properties": {
+    "raw_tx": {
+      "type":"string",
+      "description": "raw tx details"
```

Comment:
A more accurate description would be something like "A hex encoded serialized transaction"

Reply:
Updated description.